.env.example

```
@@ -7,3 +7,8 @@
   # Ollama (only needed if not using localhost:11434 default)
   OLLAMA_API_BASE=http://localhost:11434
   EVOSEAL_DEFAULT_PROVIDER=ollama
+  # Local co-evolution models (CPU path). Coder writes, reviewer critiques.
+  # Pull with: ollama pull deepseek-coder-v2:16b-lite-instruct-q8_0
+  #            ollama pull qwen2.5-coder:7b-instruct-q6_K
+  EVOSEAL_CODER_MODEL=deepseek-coder-v2:16b-lite-instruct-q8_0
```

dotenv-linter (4.0.0)

[warning] [LeadingCharacter] Invalid leading character detected

(LeadingCharacter)